src/Storages/MergeTree/registerStorageMergeTree.cpp

```diff
@@ -454,6 +455,13 @@ static StoragePtr create(const StorageFactory::Arguments & args)
                     ErrorCodes::BAD_ARGUMENTS);
             --arg_cnt;
         }
+
+        if (!tryGetIdentifierNameInto(engine_args[arg_cnt - 1], merging_params.sign_column))
```

Comment:
What's the purpose of it? type validation?

Reply:
More a column naming checker if I understand it correctly